### PR TITLE
Added option to seed random.

### PR DESCRIPTION
**Pull request by jdhodges**
#593.

### CODE REVIEW

**Comment by jdhodges** [comment]
Append `?seed=` to url to use.

File: Gruntfile.js [hunk 1]
```diff
@@ -115,7 +117,14 @@ module.exports = function(grunt) {
...
                     expand: true
-                }]
+                },
+                {
+                    cwd: 'node_modules/seedrandom',
+                    src: ['seedrandom.js'],
+                    dest: 'dist/assets/js',
+                    expand: true
+                }
+                ]
```
Comment (jleft) [inline Gruntfile.js]: Move `]` to line above (as before)

File: src/assets/js/app.js [hunk 2]
```diff
@@ -447,6 +447,11 @@ export default function() {
             throw new Error('An element must be specified when running the application.');
         }
 
+        var seed = window.location.search.split('seed=')[1];
```
Comment (jleft) [inline src/assets/js/app.js]: Please can these changes be moved to main.js, before we run the app?

**Comment by jleft** [comment]
Thanks @jdhodges - LGTM :+1: 

I've continued this in #598 with the updated shrinkwrap

**Pull request closed without merging**